### PR TITLE
Prevent clearing of worksheet when updating.

### PR DESCRIPTION
**Pull request by BrynCooke**

### CODE REVIEW

File: google_sheet/GoogleSheet.py [hunk 1]
```diff
@@ -25,35 +28,45 @@ def __init__(cls, google_sheet_name, worksheet_name):
...
-				curr_row.append(data_dict[query])
-			list_of_rows.append(curr_row)
-		# at this point, we have our list of rows. Write them to Google sheet
-                for row in list_of_rows:
-			print row
-                        self.worksheet.append_row(row)
-		self.worksheet.append_row([])
-		# allow user to see last time the worksheet was updated
+			for idx, data_dict in enumerate(list_of_data_dicts):
+				self.get_cell(current_row_number, idx + 2).value = data_dict[query]
```
Comment (ShaunakDas88) [inline google_sheet/GoogleSheet.py]: I am guessing that idx indexing starts at 0, hence the idx + 2 ( idx+1 = column of queries)?

File: google_sheet/GoogleSheet.py [hunk 2]
```diff
@@ -25,35 +28,45 @@ def __init__(cls, google_sheet_name, worksheet_name):
...
+
+		# clear the remainder of the cells
+		for cell in self.cells:
+			if cell.row > current_row_number and cell.col < len(column_headers) + 1:
+				cell.value = ''
```
Comment (ShaunakDas88) [inline google_sheet/GoogleSheet.py]: This is the case that we had more rows (i.e. queries) in the previous run than the most current run. Should we worry about the case that the previous update had seeded more columns than the most recent update? I suppose for our purposes, we have been using the same number of columns for a fixed spreadsheet

Comment (ShaunakDas88) [inline google_sheet/GoogleSheet.py]: I guess you have been wanting to preserve the "Is Faster" and " Percent difference" columns.

**Comment by ShaunakDas88** [comment]
+1

File: google_sheet/GoogleSheet.py (hunk 1 above)
Comment (BrynCooke) [inline google_sheet/GoogleSheet.py]: Sheet indexes start at 1. So just skipping the first column.

**Pull request merged**